lib/jekyll/plugin_manager.rb

```diff
@@ -27,7 +27,7 @@ def require_gems
       site.gems.each do |gem|
         if plugin_allowed?(gem)
           Jekyll.logger.debug("PluginManager:", "Requiring #{gem}")
-          require gem
+          Jekyll::External.require_with_graceful_fail([gem])
         end
```

Comment:
We make no guarantees, so they should not rely on it. Is there a debug message in require_with_graceful_fallback?